grr/server/grr_response_server/gui/ui/tsconfig.json

```diff
@@ -29,7 +29,9 @@
     ],
     "paths": {
       "@app/*": ["./*"]
-    }
+    },
+    "resolveJsonModule": true,
```

Comment:
Please see my comment above about the direct JSON import. Unfortunately we can't use this feature.